### PR TITLE
Hugo: add link to sitemap in robots.txt

### PR DESCRIPTION
**Pull request by anne-usmedia**
- Created robots.txt to override the autogenerated robots.txt (by Hugo)
- Added the sitemap link to the robots.txt
- https://gohugo.io/templates/robots/
- To test: /robots.txt

For https://linear.app/usmedia/issue/CUE-293

### CODE REVIEW

File: hugo/layouts/robots.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+User-agent: *
```
Comment (JorindeUsMedia) [inline hugo/layouts/robots.txt:1]: According to google it would be best to also add 
Allow: / 
on the 2nd line

File: hugo/layouts/robots.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+Sitemap: https://alpha.cuelang.org/sitemap.xml
```
Comment (JorindeUsMedia) [inline hugo/layouts/robots.txt:3]: Can we make this coming from the hugo config instead of hardcoded using https://gohugo.io/functions/absurl/ ?

**Review by JorindeUsMedia — CHANGES_REQUESTED** [review]

**Review by JorindeUsMedia — APPROVED** [review]

**Review by jpluscplusm — APPROVED** [review]

**Comment by jpluscplusm** [comment]
Imported as https://review.gerrithub.io/c/cue-lang/cuelang.org/+/1167807

**Comment by jpluscplusm** [comment]
Closed via 773492ee1fe2da4a3dcfb22384a6d1d38b9c05b2.

**Pull request closed without merging**